Documentation/Books/Manual/Analyzers/README.md

```diff
@@ -0,0 +1,208 @@
+# Analyzers powered by IResearch
+
+# Background
+
+A concept of value "analysis" that is meant to break up a given value into
```

Comment:
this sentence sounds a bit wonky

Reply:
that's what you get for copy-pasting from another place in the documentation
done